src/ray/core_worker/actor_handle.h

```diff
@@ -99,6 +124,8 @@ class ActorHandle {
   ObjectID actor_cursor_ GUARDED_BY(mutex_);
   // Number of tasks that have been submitted on this handle.
   uint64_t task_counter_ GUARDED_BY(mutex_) = 0;
+  absl::flat_hash_map<TaskID, uint64_t> callers_start_at_ GUARDED_BY(mutex_);
+  absl::flat_hash_map<TaskID, uint64_t> completed_task_counter_ GUARDED_BY(mutex_);
```

Comment:
Do we ever garbage collect these?

Reply:
Only when the actor handle is garbage collected...We could do it for individual entries too, but right now the ref counting for actor handles doesn't differentiate between callers.